src/index.html

```diff
@@ -1,13 +1,479 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+  <head class="page">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Museum</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/favicon.png">
+
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@300;400;500;600;700&family=Montserrat:wght@300;400;500;600;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__content">
+
+    <header class="header page__header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__image-container">
+            <a href="#" class="logo logo--header">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="logo"
+                >
```

Comment:
Need to fix code style everywhere
```suggestion
              <img
                class="logo__image"
                src="./images/logo.svg"
                alt="logo"
              >
```